### PR TITLE
Added a Handy Cheat Sheet for Kubernetes

### PR DESCRIPTION
**Pull request by Akshama**
## Hacktoberfest notes:

- due to volume of submissions, we may not be able to review PRs that do not pass tests and do not have informative titles.
- please read our [contributing guidelines](/CONTRIBUTING.md)
- be sure to check the output of Travis-CI for linter errors
- if this is your first open source contribution, make sure it's not your last!

## What does this PR do?
Add Resource(s) | Remove Resource(s) | Add info | Improve Repo

### Checklist:
- [ ] Not a duplicate
- [ ] Included author(s) if appropriate
- [ ] Lists are in alphabetical order
- [ ] Needed indications added (PDF, access notes, under construction)

### CODE REVIEW

File: free-programming-cheatsheets.md [hunk 1]
```diff
@@ -5,12 +5,12 @@
 * [Git](#git)
 * [HTML / CSS](#html--css)
 * [JavaScript](#javascript)
-  * [Vue.js](#vuejs)
+* [Vue.js](#vuejs)
```
Comment (SethiPandi) [inline free-programming-cheatsheets.md:8]: Keep keep this with two spaces before. The this a nested list in the table of contents.

Notice how Vue is an h4 (####), not an h3 (###).

```suggestion
  * [Vue.js](#vuejs)
```

**Review by SethiPandi — CHANGES_REQUESTED** [review]
Could you please ensure all lists and categories are in alphabetical order.

You can get more guidance on the formatting and conventions of this repository by checking out the [contribution guide](https://www.github.com/EbookFoundation/free-programming-books/tree/master/CONTRIBUTING.md).

You can also click on the failed checks to get more information on any check failures. You'll be able to see exactly what was wrong and how to fix it!

**Comment by Akshama** [comment]
Thanks for the suggestions. Made the necessary changes and committed again. Please check the latest commit.

**Comment by SethiPandi** [comment]
> Thanks for the suggestions. Made the necessary changes and committed again. Please check the latest commit.

It looks good to me now at least. \\(^-^)/

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**